src/visualization/CMakeLists.txt

```
@@ -102,6 +102,11 @@ if(IDYNTREE_USES_MESHCATCPP)
     target_link_libraries(${libraryname} PRIVATE MeshcatCpp::MeshcatCpp)
 endif()
 
+if(IDYNTREE_USES_ASSIMP)
+    target_compile_definitions(${libraryname} PRIVATE IDYNTREE_USES_ASSIMP)
+    target_link_libraries(${libraryname} PRIVATE assimp)
```

```suggestion
    target_link_libraries(${libraryname} PRIVATE assimp::assimp)
```

See https://github.com/robotology/idyntree/blob/07fffef70504d7e610d9466b4c3d12fd8ce610fc/src/solid-shapes/CMakeLists.txt#L28 as a reference. The logic there is more complex, but fortunately as we dropped Ubuntu 18.04 (see https://github.com/robotology/robotology-superbuild/pull/1527), we can just use the simple logic of linking to `assimp::assimp`

---

done in https://github.com/mebbaid/idyntree/commit/0b6c88044c4cfc31f6f09ebaf18851fe83ec6377